meta-mender-commercial/conditional/mender-monitor/mender-monitor.inc

```diff
@@ -2,7 +2,7 @@ inherit mender-licensing
 
 inherit systemd
 
-RDEPENDS:${PN} = "bash mender-client (>= 2.5) lmdb"
+RDEPENDS:${PN} = "bash mender-auth lmdb"
```

Comment:
Modifying this here in the `.inc` file, and similarly for `mender-gateway`, `mender-connect`, `mender-configure` (possibly others), it will misbehave for the tagged recipes. The best would be to override `RDEPENDS:${PN}` in the recpies for the already released software.

Edit: looking at your last commit, I can see that this is intentional from your end. I think it is incorrect:  an already released addons should not depend on the latest client (specially when the default is still the old one!), rather it should depend on any client. It should be `mender-auth | mender-client (>= 2.5)`, somehow.

Reply:
Yes, but this should be fine due to the [`RPROVIDES` of the mender-client recipe](https://github.com/mendersoftware/meta-mender/blob/4676d0c25750067a77de7adf14f25bf87da64d1e/meta-mender-core/recipes-mender/mender-client/mender-client-go.inc#L19).